### ISSUE TITLE
Supporting XBF version 2.1

### ISSUE THREAD

**Issue by gia0** [WalkingCat/XbfDump#2]
Great thank you for your code sharing,
but now i have i troubles with elaborating XBF version 2.1. 
Do you plan to extend your code for supporting 2.1 version?

Thanks!

**Comment by WalkingCat** [comment]
no, since its already be worked on by @misenhower  😉 https://github.com/misenhower/XbfAnalyzer

**Comment by gia0** [comment]
Great thank you!
